lib/membrane_opus/parser.ex

```diff
@@ -79,8 +80,13 @@ defmodule Membrane.Opus.Parser do
     {delimitation_processor, self_delimiting?} =
       Delimitation.get_processor(state.delimitation, state.input_delimitted?)
 
-    case maybe_parse(state.buffer <> data, state.input_delimitted?, delimitation_processor) do
-      {:ok, buffer, packets, channels} ->
+    case maybe_parse(
+           state.buffer <> data,
+           state.pts,
+           state.input_delimitted?,
+           delimitation_processor
+         ) do
+      {:ok, buffer, pts, packets, channels} ->
         packet_actions =
           if length(packets) > 0 do
             [
```

Comment:
I know that it has been there previously, but is there a reason why we send the caps each time we parse buffers?